sources/api/early-boot-config/src/provider/vmware.rs

```diff
@@ -190,21 +183,22 @@ mod error {
         #[snafu(display("Unable to read input file '{}': {}", path.display(), source))]
         InputFileRead { path: PathBuf, source: io::Error },
 
-        #[snafu(display("Error serializing TOML to JSON: {}", source))]
-        SettingsToJSON { source: serde_json::error::Error },
+        #[snafu(display(
+            "Invalid (non-utf8) output from base64 string '{}': {}",
+            base64_string,
+            source
+        ))]
+        InvalidUTF8 {
+            base64_string: String,
+            source: std::str::Utf8Error,
+        },
 
-        #[snafu(display("Error parsing TOML user data: {}", source))]
-        TOMLUserDataParse { source: toml::de::Error },
+        #[snafu(context(false))]
+        SettingsToJSON { source: crate::settings::Error },
```

Comment:
I'm less confident about removing context from this error variant.  VMware has multiple sources of user data.  Is it going to be clear which failed?

Reply:
Added some additional context